### DISCUSSION TITLE
DAI 2023-2024 - Practical work 2 - Piemontesi Gwendal et Trueb Guillaume

### DISCUSSION THREAD

**Discussion by tasty-orange** [heig-vd-dai-course/heig-vd-dai-course#142] [Show and tell]
Nous voulons réaliser une bataille naval simple à deux joueurs.

**Comment by grinlemon** [comment]
The link to the repository : https://github.com/gwendalpiemonte/SheepMaBoat
This is our final version commit hash : 0ba1dc8c8f2211852f4fc97fee8dbb402e0d926d
